Makefile

```diff
@@ -226,6 +226,13 @@ urn:
 	@printf "⚱\n\n";
 	@$(MAKE) run;
 
+docker-build:
+	docker build -t wp-calypso .
+
+docker-run:
+	docker run -it --name wp-calypso --rm -p 80:3000 -e NODE_ENV='wpcalypso' -e CALYPSO_ENV='wpcalypso' wp-calypso
```

Comment:
`echo "running wp-calypso docker image at localhost:3000"` ?

What do you think the message should be?